pkg/apb/deprovision.go

```diff
@@ -24,19 +23,14 @@ func Deprovision(instance *ServiceInstance, log *logging.Logger) error {
 		return err
 	}
 
-	if err = client.PullImage(instance.Spec.Name); err != nil {
-		return err
-	}
-
```

Comment:
@jmrodri [ImagePullPolicy setting on](https://github.com/openshift/ansible-service-broker/blob/master/pkg/apb/client.go#L194) Pod creation will handle this.